adds persistence.storageClass and limits to postgress helm install

coreywan opened the pull request:
##### SUMMARY
<!--- Describe the change, including rationale and design decisions -->
This commit adds some options to the helm deployment. Specifically, it allows the selection of the storageclass, and setting of the resource limits.
<!---
If you are fixing an existing issue, please include "related #nnn" in your
commit message and your description; but you should still explain what
the change does.
-->

##### ISSUE TYPE
<!--- Pick one below and delete the rest: -->
 - Feature Pull Request

##### COMPONENT NAME
<!--- Name of the module/plugin/module/task -->
 - Installer

##### AWX VERSION
<!--- Paste verbatim output from `make VERSION` between quotes below -->
```
awx: 2.1.2
```


##### ADDITIONAL INFORMATION
<!---
Include additional information to help people understand the change here.
For bugs that don't have a linked bug report, a step-by-step reproduction
of the problem is helpful.
  -->

<!--- Paste verbatim command output below, e.g. before and after your change -->
```

```

coreywan committed:
* adds persistence.storageClass and limits to postgress helm install
* adds new variables to the inventory

Signed-off-by: Corey Wanless <corey.wanless@wwt.com>

matburt commented:
Excellent, a few things though... can you sign your PR as requested in the contributing doc?

I'd also like to surface these as commented out items in the reference inventory file and in the readme installer docs.

coreywan commented:
@matburt , I believe I made the changes you requested.

matburt commented:
Excellent, I just need you to sign you initial PR message as per the DCO mentioned in the contributing docs.

coreywan commented:
@matburt , I apologize. I don't fully understand your ask. When I updated the inventory, I squashed the commits together and signed off that commit.  I don't understand what I need to do to sign off on a PR.

matburt commented:
Ahh I do see it now, I'll check in on this tomorrow.

ryanpetrello commented:
@matburt did you mean to code approve this?
